src/components/registerV2/backupPassphrase.js

```diff
@@ -98,20 +91,29 @@ class BackupPassphrase extends React.Component {
                 </Tooltip>
 
               </h2>
-              <p className='option-value'>{account.passphrase}</p>
+              <div className={`${styles.inputs} ${grid.row} passphrase`}>
+                {account.passphrase.split(' ').map((value, i) => (
+                  <span key={i} className={`${grid['col-xs-2']}`}>
+                    <InputV2
+                      disabled={true}
```

Comment:
maybe use the readonly mode instead of disabled? so the cursor changes to the text one.